tests/testthat/test_inspectDifferences.R

```diff
@@ -40,21 +40,31 @@ test_that("Compare two different GADSdat objects",{
 })

```

Comment:
Please add to this section a test where argument "other_varName" is used

Reply:
Agree and added the requested test (which also captured the previous issue with `other_varName`).